src/modules/dashboard/components/ActionsPage/staticMaps.ts

```diff
@@ -71,4 +87,23 @@ export const EVENTS_REQUIRED_FOR_ACTION: ActionsEventsMap = {
   [ColonyActions.MoveFunds]: [
     ColonyAndExtensionsEvents.ColonyFundsMovedBetweenFundingPots,
   ],
+  [ColonyActions.MintTokens]: [ColonyAndExtensionsEvents.TokensMinted],
+};
+
+/*
+ * Which details display for which type
+ */
+
+export const DETAILS_FOR_ACTION: ActionsDetailsMap = {
+  [ActionPageDetails.fromDomain]: [
+    ColonyActions.Payment,
+    ColonyActions.MoveFunds,
+  ],
+  [ActionPageDetails.toDomain]: [ColonyActions.MoveFunds],
+  [ActionPageDetails.toRecipient]: [ColonyActions.Payment],
+  [ActionPageDetails.amount]: [
+    ColonyActions.Payment,
+    ColonyActions.MoveFunds,
+    ColonyActions.MintTokens,
+  ],
```

Comment:
This should be reversed. The details should based on actions, **not** the actions on the details.

Eg:
```
[ColonyActions.Payment]: [
  ActionPageDetails.fromDomain,
  ActionPageDetails.toRecipient,
  ActionPageDetails.amount
],
```